### ISSUE TITLE
The first field should be focused when editor dialog is opened

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-crud#129]
The first field in the form should have keyboard focus when the dialog is opened.

Small devices with software keyboards should preferably be excluded.

**Comment by manolo** [comment]
Mostly depends on https://github.com/vaadin/vaadin-overlay/issues/23 . Otherwise we need some hack

**Comment by Artur-** [comment]
vaadin/vaadin-overlay#23 was fixed some months ago

**Comment by web-padawan** [comment]
See also https://github.com/vaadin/vaadin-dialog/issues/99#issuecomment-409823306
